docs/tutorials/basics/60_methods.md

```diff
@@ -107,3 +107,60 @@ say_hello 3
 
 Overloading isn't defined just by type restrictions. The number of arguments as well as named arguments are also
 relevant characteristics.
+
+## Return value
+
+Methods may also return a value:
+
+```crystal-play
+def adds_2(n : Int32 = 0)
+  n + 2
+end
+
+puts adds_2 40
+```
+
+The keyword `return` is not necessary and methods will always return the result of executing the last line.
+
+The following example usest the `return` keyword and _implicit_ `return`:
```

Comment:
```suggestion
The following example illustrates the use of an _explicit_ and an _implicit_ `return`:
```
Even better, join it to the previous paragraph, with something like "An _explicit_ return allows us to break earlier from the function. The following..."